javascript/javascript1/week4/voiceassistant.js

```diff
@@ -0,0 +1,93 @@
+let inputname = "";
+let todo = [];
+function getReply(command)
+{
+    const arraywords = command.toLowerCase().split(" ");
+    if (arraywords.includes("hello"))
+    {
+        inputname = arraywords [arraywords.length-1];
+        return `Nice to meet you ${inputname}` ;
+    }
+    else if (command.includes("What is my name")) 
+    {
+        return `Your name is ${inputname}` ;
+    }
+    else if (command.includes("Add fishing"))
+    {
+       todo.push(command);
+       return `fishing added to your todo` ;
+    }
+    else if (command.includes("Add singing"))
```

Comment:
Try to make a more dynamic solution to be able to get todo time whatever is it.
hint
```js
    const index1 = _command.indexOf('add ');
    const index2 = _command.indexOf(' to my todo');
    if (index1 !== -1 && index2 !== -1) {
      const todo = _command.substring(index1 + 'add '.length, index2);
}
```
Do the same for removing